test-suites/gutenberg/sanity-tests.md

```diff
@@ -67,26 +67,30 @@ Spacer-1
 
 Button-1
 
-- [ ] Button block - Custom colors are rendered OK for text - steps
-- [ ] Button block - Custom colors are rendered OK for background - steps
-- [ ] Button block - Gradient colors rendered OK for background - steps
+- [ ] Button block - Custom colors are rendered OK for text - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/button.md#tc011)
```

Comment:
Could you also use same titles here with the `test-cases/gutenberg/button.md` ? Otherwise when you follow the "steps" link it can be confusing.